### ISSUE TITLE
add additional modules to system image

### ISSUE THREAD

**Issue by andrewning** [JuliaPy/pyjulia#311]
Is there a way I can some of my own packages to the custom system image?

**Comment by stevengj** [comment]
Duplicate of #310?

**Comment by andrewning** [comment]
I don't think it was a duplicate.  #310 was a problem (perhaps a bug) in just building the system image that @tkf provided a workaround for.  In this issue I was asking about whether there was a way to customize the system image that is built (adding additional packages).  I figured out I could do it by modifying precompile.jl but it would be nice if there was a command line option to pass in a file rather than manually modify the package.

**Comment by tkf** [comment]
There is no easy way to do this at the moment.  Maybe first create a system image for PyJulia, start `julia-py` command with this system image, and then use PackageCompiler to build a new system image with more Julia packages?
